docs/ballerina-by-example/examples/http-access-logs/http-access-logs.bal.server.sh

```diff
@@ -0,0 +1,10 @@
+#Run the service by setting '-B[accesslog.http].logto=console' to log to console
+$ ballerina run http-access-logs.bal -B[accesslog.http].logto=console
```

Comment:
Can't we come up with another name for the log output location?

Reply:
Do you mean to change the name console to a different value? Can you suggest any?

Reply:
We'll get back to this later, with the config API changes